### ISSUE TITLE
Running into catastrophic failure of the client side utilities on some urls.

### ISSUE THREAD

**Issue by msimr** [casperjs/casperjs#1091]
I'm running into some quite strange crashes with client utils on some urls.
exemple: http://www.toysrus.com/product/index.jsp?productId=29463186

```
[phantom] Successfully injected Casper client-side utilities
Error: "TypeError: undefined is not a constructor (evaluating 'new window.ClientUtils')"
Error: "ReferenceError: Can't find variable: __utils__"
```

**Comment by paazmaya** [comment]
Could you provide a minimal working example?

**Comment by khodya** [comment]
Hi, I have the same issue when I try to work with popups.
This is 100% reproducible behavior for the specific popup which I try to open. Another my script that opens a different popup does not reproduce this behavior. 

I added an event handler for the 'step.added' event which prints the step added. Here is a fragment from debug output:
```
step added: function _step() {
        var popupPage = this.popups.find(popupInfo);
        if (!utils.isFunction(then)) {
            throw new CasperError("withPopup() requires a step function.");
        }
        // make the popup page the currently active one
        this.page = popupPage;
    }
step added: function () {}
step added: function _step() {
        // revert to main page
        this.page = this.mainPage;
    }
[info] [phantom] Step anonymous 14/16: done in 17492ms.
[info] [phantom] Step _step 15/19 <url link> (HTTP 200)
[info] [phantom] Step _step 15/19: done in 17514ms.
[debug] [phantom] Successfully injected Casper client-side utilities
TypeError: undefined is not a constructor (evaluating 'new window.ClientUtils')

  undefined:2
  :3
[debug] [phantom] Successfully injected Casper client-side utilities
TypeError: undefined is not a constructor (evaluating 'new window.ClientUtils')

  undefined:2
  :3

```

**Comment by istr** [comment]
@khodya Please have a look at the current open issues. There are some  ongoing discussions about the popup issues, esp. with newer versions of PhantomJS. If you identify a matching issue, please reference it here, so we can see if it is a duplicate. Thank you.

**Comment by BIGjuevos** [comment]
Popup issues were resolved in 1.1.0.  @khodya please let us know if it solves your problem.

**Comment by kensoh** [comment]
@msimr @khodya it seems that a few fixes had been committed for popups since then. Closing this issue for now but do ping if the issue still persists for you.

I'm a CasperJS user for couple of years. Browsing through open issues now to see if anything I can help with. If you have a script and website that can replicate the issue I can try to run it.
